shell/components/nav/TopLevelMenu.vue

```diff
@@ -203,7 +207,7 @@ export default {
       const cluster = this.clusterId || this.$store.getters['defaultClusterId'];
 
       // TODO plugin routes
-      const entries = this.activeProducts.map((p) => {
+      const entries = this.$store.getters['type-map/activeProducts']?.map((p) => {
```

Comment:
Because when I mock it gives an error that cannot find `type-map/` no matter what, so since it's the only case of the entire app to be mapped like that, I just changed it back to speed up things.
I have to investigate on how to mock `mapGetters()` opposed to the way I do it now, although it will be different on Vue3, so I did not want to waste time on an obsolete concept.